src/main/kotlin/presentation/ui/ClothesSuggesterByIpCli.kt

```diff
@@ -0,0 +1,74 @@
+package org.damascus.presentation.ui
+
+import kotlinx.coroutines.*
+import org.damascus.domain.model.Cloth
+import org.damascus.domain.usecase.GetWeatherByIpUseCase
+import org.damascus.domain.usecase.SuggestClothesUseCase
+import org.damascus.presentation.io.ConsoleDisplay
+import presentation.TerminalColor
+import presentation.ui.UILauncher
+import presentation.withStyle
+
+class ClothesSuggesterByIpCli(
+    private val printer: ConsoleDisplay,
+    private val getWeatherByIpUseCase: GetWeatherByIpUseCase,
+    private val suggestClothesUseCase: SuggestClothesUseCase,
+) : UILauncher {
+
+    private var suggestedClothes: List<Cloth>? = null
```

Comment:
This variable does nothing, just declared to get assigned later and no use after.